docs/config.md

```diff
@@ -1455,41 +1455,49 @@ Setting this value to zero will disable the limit entirely.
 
 ## Ways to specify `jj` config: details
 
-### User config file
+### User config files
 
-An easy way to find the user config file is:
+An easy way to find the user config file/directory is:
 
 ```bash
 jj config path --user
 ```
 
-The rest of this section covers the details of where this file can be located.
+On all platforms, the user's global `jj` configurations are loaded in the following
+precedence order (with later configs overriding earlier ones):
 
-On all platforms, the user's global `jj` configuration file is located at either
-`~/.jjconfig.toml` (where `~` represents `$HOME` on Unix-likes, or
-`%USERPROFILE%` on Windows) or in a platform-specific directory. The
-platform-specific location is recommended for better integration with platform
-services. It is an error for both of these files to exist.
+- `$HOME/.jjconfig.toml` (not recommended)
```

Comment:
nit: remove `(not recommended)`? It sounds a bit stronger than just saying "platform location is recommended for `<reasons>`."

Reply:
I wanted to draw attention to it in the list which is what most people will look at---I imagine that some might skip over the recommendation in the text.

Would you be opposed to adding `(preferred)` next to the `config.toml` (or changing `not recommended` to `not preferred`)?

Reply:
`(preferred)` sounds good to me.